### PR TITLE
[Doc-90] List YB Voyager as a product

### PR DESCRIPTION
**Pull request by aishwarya24**
Summary: 
- Added YB Voyager as an entry in the products dropdown 
- Added a homepage for YB Voyager and all the items in it in the left nav following the structure similar to YBA and YBM : https://deploy-preview-18639--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/
- Added netlify redirects and updated menu.toml

### CODE REVIEW

**Comment by ymahajan** [comment]
Instead of 'Use YugabyteDB Anywhere' it should be "Use YugabyteDB Voyager" on the home page.

**Comment by ddhodge** [comment]
Landing and overview page content needs to be finalized/worked on, but the structure is more or less what you are looking for @ymahajan?

**Comment by ymahajan** [comment]
> 

Correct. We can keep working on the home page later.

**Comment by ymahajan** [comment]
@aishwarya24 - It looks good, we should commit this.

**Review by ymahajan — APPROVED** [review]

File: docs/content/preview/yugabyte-voyager/_index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    identifier: yugabyte-voyager
+    parent: yugabytedb-voyager
+    weight: 99
+breadcrumbDisable: true
+resourcesIntro: Quick Links
+resources:
+  - title: What's new
+    url: /preview/yugabyte-voyager/release-notes/
+  - title: Download
+    url: https://download.yugabyte.com/migrate#
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-voyager/_index.md:21]:
```suggestion
    url: https://download.yugabyte.com/migrate
```

Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/_index.md:21]: The url appends a slash  (/) after the # making it `https://download.yugabyte.com/migrate#/`- so it points to local install instead incorrectly.

File: docs/content/preview/yugabyte-voyager/overview.md [hunk 2]
```diff
@@ -45,32 +48,37 @@ YugabyteDB Voyager supports migrating schema and data from your existing RDBMS,
...
+| [Analyze schema ](../migrate-steps/#analyze-schema) | Generate a *Schema&nbsp;Analysis&nbsp;Report* using the `yb-voyager analyze-schema` command. The report suggests changes to the PostgreSQL schema to make it appropriate for YugabyteDB. |
+| [Modify schema](../migrate-steps/#manually-edit-the-schema) | Using the report recommendations, manually change the exported schema. |
+| [Export data](../migrate-steps/#export-data) | Dump the source database to the target machine (where yb-voyager is installed), using the `yb-voyager export data` command. |
+| [Import schema](../migrate-steps/#import-schema) | Import the modified schema to the target YugabyteDB database using the `yb-voyager import schema` command. |
+| [Import data](../migrate-steps/#import-data) | Import the data to the target YugabyteDB database using the `yb-voyager import data` command. |
+| [Import&nbsp;indexes/triggers](../migrate-steps/#import-indexes-and-triggers) | Import indexes and triggers to the target YugabyteDB database using the `yb-voyager import schema` command with an additional `--post-import-data` flag. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-voyager/overview.md:76]:
```suggestion
| [Import&nbsp;indexes and triggers](../migrate-steps/#import-indexes-and-triggers) | Import indexes and triggers to the target YugabyteDB database using the `yb-voyager import schema` command with an additional `--post-import-data` flag. |
```

File: docs/content/preview/yugabyte-voyager/overview.md [hunk 3]
```diff
@@ -45,32 +48,37 @@ YugabyteDB Voyager supports migrating schema and data from your existing RDBMS,
...
-- Import the schema to the target YugabyteDB database using the [`yb-voyager import schema`](migrate-steps/#import-schema) command.
-- Import the data to the target YugabyteDB database using the [`yb-voyager import data`](migrate-steps/#import-data) command.
-- Import indexes and triggers to the target YugabyteDB database using the [`yb-voyager import schema`](migrate-steps/#import-indexes-and-triggers) command with an additional `--post-import-data` flag.
+A typical migration workflow using yb-voyager consists of the steps shown in the following illustration:
 
 ![Migration workflow](/images/migrate/migration-workflow.png)
 
+| Step | Description |
+| :--- | :---|
+| [Install yb-voyager](../install-yb-voyager/#install-yb-voyager) | yb-voyager supports RHEL, CentOS, Ubuntu, MacOS, and Docker-based installations. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-voyager/overview.md:67]:
```suggestion
| [Install yb-voyager](../install-yb-voyager/#install-yb-voyager) | yb-voyager supports RHEL, CentOS, Ubuntu, macOS, Airgapped and Docker-based installations. |
```

**Review by aishwarya24 — COMMENTED** [review]
some comments

File: docs/content/preview/yugabyte-voyager/_index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  - title: yugabyte.com
+    url: https://www.yugabyte.com/voyager/
+---
+
+Use YugabyteDB Voyager to manage end-to-end database migration, including cluster preparation, schema migration, and data
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-voyager/_index.md:26]: do we want to add links to YBM and YBA , and core DB in this intro ?

**Pull request merged**